extensions/amp-animation/amp-animation.md

```diff
@@ -27,7 +27,7 @@ limitations under the License.
   </tr>
   <tr>
     <td width="40%"><strong>Required Script</strong></td>
-    <td><code>&lt;script async custom-element="amp-animation" src="https://cdn.ampproject.org/v0/amp-animation-0.1.js">&lt;/script></code></td>
```

Comment:
Oh, that was a test change that I didn't mean to commit :) I hadn't yet published this PR for review, and was merely testing a definitely-broken script tag link to make sure it got filtered.

I'll republish this PR without that change.